### PR TITLE
add dark mode

### PR DESCRIPTION
**Pull request by Velua**

### CODE REVIEW

**Comment by brandonfancher** [comment]
Quality! Love it. Will review shortly.

File: packages/webapp/src/pages/induction/[id].tsx [hunk 1]
```diff
@@ -49,12 +49,11 @@ export const InductionDetailsPage = () => {
 
     // TODO: Consider deriving the user's role here and return an enum of roles: INVITER, ENDORSER, INVITEE, MEMBER, EOS_USER, UNAUTHENTICATED.
     // Almost every child component of this page cares about the role of the user in relation to the invite/induction. We can pass role down.
-    const isEndorser = useMemo(() => {
-        const result = endorsements.find(
+    const isEndorser = useMemo(() => 
+         endorsements.some(
```
Comment (brandonfancher) [inline packages/webapp/src/pages/induction/[id].tsx:53]: 😍 I forget about `.some`!

**Comment by brandonfancher** [comment]
I know this is just a draft right now. Might be worth splitting the dark mode work out into a separate PR. The prior commits are good improvements and I'd love to go ahead and get those merged. And I think dark mode deserves its own PR.

**Comment by Velua** [comment]
@brandonfancher Yeah was thinking that as I was doing it... will do!

**Comment by Velua** [comment]
@brandonfancher added now in #201

**Comment by Velua** [comment]
Closing in favour of #213

**Pull request closed without merging**